prow/config/external_plugins_config.yaml

```diff
@@ -930,6 +930,10 @@ ti-community-contribution:
       - pingcap/ticdc
       - pingcap/br
       - chaos-mesh/chaos-mesh
+      - pingcap/docs
```

Comment:
Need to add the corresponding event config, see: https://github.com/ti-community-infra/configs/blob/3bd74980b1dc6a11525d559dacc3515400311037/prow/config/plugins.yaml#L336